docs/spec_draft.md

```diff
@@ -277,6 +280,46 @@ Produces a `result` tensor from a constant `value`.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.cosine
+
+`stablehlo.cosine(operand) -> result`
+
+### Semantics
+
+Performs element-wise cosine operation on `operand` tensor and produces a
+`result` tensor, according to the IEEE-754 specification. Numeric precision is
```

Comment:
`implementing to the `cos` operation from the IEEE-754 specification.`
Same for other ops below.

Reply:
Done.